projects/js-packages/components/changelog/update-use-workspace-imports-for-js-tools

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Use workspace for imports from js-tools
```

Comment:
Since it's only monorepo tooling that's changing, nothing affecting the actual package, perhaps we should do it like this to avoid a pointless changelog entry
```suggestion
Comment: Use workspace for imports from js-tools. Monorepo tooling only, no change to the package.

```
Same for all the other change entry files.

Reply:
Done, thanks